build.xml

```diff
@@ -206,6 +206,10 @@
         </jacoco:report>
     </target>
 
+    <target name="extensions_test" depends="build">
```

Comment:
Add a comment about the What.  For instance, something like what you said in the pull request "This will execute extension tests directly from the main directory, with ./refine extensions_test ...blah"

Reply:
sure, done (I've left out the bit about ./refine because people can also use ant directly)